code/index.html

```diff
@@ -20,24 +20,37 @@
         alt="Guess Who"
       />
 
-      <h1>Does the person have</h1>
+      <h1>Does the person have...?</h1>
 
       <select id="questions">
         <optgroup label="hair">
+          <option>START HERE!!!</option>
```

Comment:
Haha, I love your use of exclamation points. It makes it superclear where to start this game!